apps/dolly-frontend/src/main/js/src/components/bestilling/sammendrag/kriterier/BestillingKriterieMapper.tsx

```diff
@@ -55,7 +55,20 @@ const expandable = (
 	objects,
 })
 
-const mapBestillingsinformasjon = (bestillingsinformasjon: any, data: any[], identtype: any) => {
+const mapBestillingsinformasjon = (
+	bestillingsinformasjon: any,
+	data: any[],
+	identtype: any,
+	firstIdent: string,
+) => {
+	const getTypePerson = () => {
+		const identifiserendeDigit = firstIdent?.charAt(2)
+		if (identifiserendeDigit == '0' || identifiserendeDigit == '1') {
```

Comment:
Denne håndterer ikke NPID/BOST som kan være standard, men som legger 20 på måned.
En bedre test er:
`parseInt(firstIdent?.charAt(2)) < 4`

Reply:
Aha, good to know! Skal fikse 👍